src/entries/popup/components/TransactionFee/TransactionFee.tsx

```diff
@@ -168,6 +201,12 @@ function Fee({
                   <TextOverflow weight="semibold" color="label" size="14pt">
                     {isLoading
                       ? '~'
+                      : !nativeCurrencyGasFeeSupported(chainId)
+                      ? convertToNativeGasFee(
```

Comment:
if the intention of this was using the fee as native value if the price is not available that should have been done here https://github.com/rainbow-me/browser-extension/pull/1111

that should be checking if native asset has price and if not default to crypto values

Reply:
to fix it you can change https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/hooks/useGas.ts#L54C23-L54C47

`useNativeAssetForNetwork` to `useNativeAsset`